app/assets/images/icons/x.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor" stroke-width="2">
```

Comment:
Is this icon any different from the one we use in our modals?

Reply:
I think we're using font awesome for the modal close buttons @leosoaivan. I'll create a chore to replace it, we're using these [Heroicons](https://heroicons.com/) in more places than FA now 😆